### ISSUE TITLE
Reader Interface Search Results Page Can Be Escaped By Screen Readers

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2754]
Note: This is very similar to Issue #2434, and can likely be resolved in a similar way, but it is given its own issue because it is referring to a different widget in a different context.

In Manifold's reader interface, the reader can search a text to find specific passages or annotations. This search is represented as a full page, but it is implemented as an overlay that hides all content behind it. The content below it is effectively rendered invisible, but can be reached by screen readers, causing great confusion for users.

## Expected Behavior
Unless intentionally providing assistance for non-visual users, content that has been hidden for visual users should be hidden for non-visual users as well. In this case, when content is overlaid over other content, the user should be trapped inside of the overlaid content, and not be able to reach the content behind it. 

## Current Behavior
It is possible to break out of the Search Results page using a screen reader's browse mode functionality. This page may either be escaped from the top or the bottom.

The following image shows NVDA outputting information that far exceeds the last Search Result:
![image](https://user-images.githubusercontent.com/43553368/83055800-330f1e00-a009-11ea-831e-c04fed0d7c90.png)

## Steps to Reproduce
1. Navigate to the Reader Interface
2. Navigate to a project with readable and searchable text
3. Navigate into a text contained in that project
4. Navigate to and activate the search button in the menu
5. Enter any search criteria into the edit field and submit the search
6. Navigate to the close button on the top of the page
7. Activate Browse Mode of a screen reader (if using NVDA, this may be done with the NVDA Modifier Key + Space)
6. Using the up arrow key, continue to navigate the pane until content from reader's menu is read (though it is not visible)
7. Navigate to the last search result, and use the down arrow keys to read the content of the text (though it is not visible)

## Testing Environment
NVDA 2020.1
ESR Firefox 68.8.0 esr
